pyaedt/misc/Run_PyAEDT_Script.py_build

```diff
@@ -34,12 +34,13 @@ print("ret: " + repr(ret))
 try:
     if ret == ret.OK:
         # launch file
+        version = oDesktop.GetVersion()[2:6].replace(".", "")
         current_dir = os.path.dirname(os.path.abspath(os.path.realpath(__file__)))
         pyaedt_toolkit_dir = os.path.normpath(os.path.join(current_dir, "..", "..", r"##TOOLKIT_REL_LIB_DIR##"))
         config_file = open(
             os.path.join(pyaedt_toolkit_dir, "python_interpreter.bat"), "r"
         )  # will fail with a file not found error if it isn't here
-        python_exe = r"##PYTHON_EXE##"
+        python_exe = r"##PYTHON_EXE##" % version
```

Comment:
I can't get this working locally @kbhagat-ansys . Is it working on your side?
Maybe you were expecting something like this:
```suggestion
        python_exe = r"##PYTHON_EXE##" + version
```

Reply:
@MaxJPRey  that will not work. Refer to the Image shown in the PR description. We don't want to concatenate a string but replace a "%s" inside a string.